wercker-step.yml

```diff
@@ -1,3 +1,3 @@
 name: rails-database-yml
-version: 1.0.2
```

Comment:
@ae06710 thanks for advice :smiley: I confirmed we can add suffix to wercker step verion.  we use this branch as PR one, then I'll create Temporary branch from this one.